pkg/eth/erc20/denomination.go

```diff
@@ -1,12 +1,15 @@
 package erc20
 
 import (
+	"math"
 	"math/big"
 )
 
 // Denominate converts gwei units into ether units
-func Denominate(gwei *big.Int) *big.Int {
-	ether = new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil)
-	balance := new(big.Int).Div(gwei, ether)
-	return balance
+func Denominate(gwei *big.Int) string {
+	fbalance := new(big.Float)
+	fbalance.SetString(gwei.String())
+	ethValue := new(big.Float).Quo(fbalance, big.NewFloat(math.Pow10(18))).String()
+
+	return ethValue + " ETH"
```

Comment:
@GlenDC This function converts WEI into ETH. And will also show decimal notation.